api/swagger.yaml

```diff
@@ -4226,6 +4226,7 @@ paths:
             Filters to process on the prune list, encoded as JSON (a `map[string][]string`).
 
             Available filters:
+            - `until=<string>` Prune containers created until this timestamp.
```

Comment:
Thanks @thaJeztah. The swagger.yaml has been updated. The API takes the same format (Go, timestamp) as the cli.